packages/@coorpacademy-components/src/template/common/discipline/test/fixtures/with-associated-skills.js

```diff
@@ -0,0 +1,79 @@
+import {update, uniqueId, head} from 'lodash/fp';
+import disciplineHeader from '../../../../../molecule/discipline-header/test/fixtures/long-description';
+import disciplinePartners from '../../../../../molecule/discipline-partners/test/fixtures/more-info';
+import disciplineCTA from '../../../../../molecule/discipline-cta/test/fixtures/default';
+import disciplineScope from '../../../../../molecule/discipline-scope/test/fixtures/medias';
+import Thread from '../../../../../molecule/forum/forum-thread/test/fixtures/default';
+import ForumComment from '../../../../../molecule/forum/forum-comment/test/fixtures/default';
+
+const thread = Thread.props;
+
+const threads = [thread].map(update('id', () => uniqueId()));
+
+const {title, description, video, image, lastUpdated} = disciplineHeader.props;
+const {start, buy, startLabel, buyLabel} = disciplineCTA.props;
+const {authors} = disciplinePartners.props;
+const {content, levels} = disciplineScope.props;
+
+export default {
```

Comment:
Could you add it to some skills props to the existing templates?
`external-course.js` notably, and maybe also `default` (2/3 skills are enought. for default, could be cool with none focused)